enh(hardware-storage-netapp-ontap-snmp): Add missing service template snapmirrorlag CTOR-2084 (#4913)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-netapp-ontap-snmp.md

| Cache-Age     | HW-Storage-NetApp-Ontap-Cache-Age-SNMP-custom     | Check age in minutes of the oldest read-only blocks in the buffer cache. |
| Cp-Statistics | HW-Storage-NetApp-Ontap-Cp-Statistics-SNMP-custom | Check consistency point metrics.                                         |
| Cpu-Load      | HW-Storage-NetApp-Ontap-Cpu-Load-SNMP-custom      | Check CPU usage.                                                         |

> | Disk-Failed   | HW-Storage-NetApp-Ontap-Disk-Failed-SNMP-custom   | Check the current number of disk broken                                  |

Author: ```suggestion
| Disk-Failed   | HW-Storage-NetApp-Ontap-Disk-Failed-SNMP-custom   | Check the current number of malfunctioning disks                                 |
```

| Global-status | HW-Storage-NetApp-Ontap-Global-status-SNMP-custom | Check the overall status of the  appliance                               |

> | Nvram         | HW-Storage-NetApp-Ontap-Nvram-SNMP-custom         | Check current status of the NVRAM batteries                              |

Author: ```suggestion
| Nvram         | HW-Storage-NetApp-Ontap-Nvram-SNMP-custom         | Check the current status of the NVRAM batteries                              |
```

|:-----------------------|:-----------------------------------------------------------|:--------------------------------------------------------------------------------------|:---------:|
| Aggregates             | HW-Storage-NetApp-Ontap-Aggregates-SNMP-custom             | Check state of one or several aggregates                                              |           |
| Cluster-Nodes          | HW-Storage-NetApp-Ontap-Cluster-Nodes-SNMP-custom          | Check cluster nodes                                                                   |     X     |

> | Fan                    | HW-Storage-NetApp-Ontap-Fan-SNMP-custom                    | Check if fans are failed                                                              |           |

Author: ```suggestion
| Fan                    | HW-Storage-NetApp-Ontap-Fan-SNMP-custom                    | Check if fans have failed                                                              |           |
```

| File-System-Global     | HW-Storage-NetApp-Ontap-File-System-Global-SNMP-custom     | Check filesystem usage                                                                |     X     |

> | Ndmpsessions           | HW-Storage-NetApp-Ontap-Ndmpsessions-SNMP-custom           | Check current total of ndmp sessions opened                                           |           |

Author: ```suggestion
| Ndmpsessions           | HW-Storage-NetApp-Ontap-Ndmpsessions-SNMP-custom           | Check current total of open NDMP sessions                                           |           |
```

| Plexes                 | HW-Storage-NetApp-Ontap-Plexes-SNMP-custom                 | Check plexes                                                                          |     X     |
| Psu                    | HW-Storage-NetApp-Ontap-Psu-SNMP-custom                    | Check if power supplies have failed                                                   |           |
| Quotas                 | HW-Storage-NetApp-Ontap-Quotas-SNMP-custom                 | Check quotas                                                                          |           |
| Share-Calls            | HW-Storage-NetApp-Ontap-Share-Calls-SNMP-custom            | Check cifs and nfs calls per seconds.                                                 |           |
| Snapmirrorlag          | HW-Storage-NetApp-Ontap-Snapmirrorlag-SNMP-custom          | Check snapmirrors status and lag                                                      |           |
| Snapshot-Age-Global    | HW-Storage-NetApp-Ontap-Snapshot-Age-Global-SNMP-custom    | Check snapshot age of volumes                                                         |           |

> | Snapshot-Age-Name      | HW-Storage-NetApp-Ontap-Snapshot-Age-Name-SNMP-custom      | Check snapshot age of volumes                                                         |           |

Author: ```suggestion
| Share-Calls            | HW-Storage-NetApp-Ontap-Share-Calls-SNMP-custom            | Check CIFS and NFS calls per second                                                 |           |
| Snapmirrorlag          | HW-Storage-NetApp-Ontap-Snapmirrorlag-SNMP-custom          | Check snapmirrors status and lag                                                      |           |
| Snapshot-Age-Global    | HW-Storage-NetApp-Ontap-Snapshot-Age-Global-SNMP-custom    | Check the age of volume snapshots                                                          |           |
| Snapshot-Age-Name      | HW-Storage-NetApp-Ontap-Snapshot-Age-Name-SNMP-custom      | Check the age of volume snapshots                                                          |           |
```
